xcm/src/v0/multi_location.rs

```diff
@@ -115,6 +112,40 @@ impl MultiLocation {
 		})
 	}
 
+	/// Creates a new `MultiLocation` with 0 parents and a `Null` interior.
+	pub const fn empty() -> MultiLocation {
+		MultiLocation {
+			parents: 0,
+			interior: Junctions::Null,
+		}
+	}
+
+	/// Creates a new `MultiLocation` with the specified number of parents and a `Null` interior.
+	/// Returns an error if `parents` is greater than `MAX_MULTILOCATION_LENGTH`.
+	pub const fn with_parents(parents: u8) -> result::Result<MultiLocation, ()> {
+		if parents as usize > MAX_MULTILOCATION_LENGTH {
+			return Err(())
+		}
+		Ok(MultiLocation {
+			parents,
+			interior: Junctions::Null,
+		})
+	}
+
+	/// Creates a new `MultiLocation` with no parents and a single `Parachain` interior junction
+	/// specified by `para_id`.
+	pub const fn with_parachain_interior(para_id: u32) -> MultiLocation {
```

Comment:
```suggestion
	pub const fn from_parachain(para_id: u32) -> MultiLocation {
```
would that also work?